R/doAzureParallel.R

```diff
@@ -242,7 +256,7 @@ setVerbose <- function(value = FALSE) {
     tryCatch({
       retryCounter <- retryCounter + 1
 
-      response <- rAzureBatch::createContainer(id, raw = TRUE)
+      response <- rAzureBatch::createContainer(id, content = "text")
```

Comment:
Would it make more sense to parse a JSON response object? (Not sure if storage supports that yet)

Reply:
Storage does not support JSON response